src/redux/reducers/projectsSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const projectsSlice = createSlice({
```

Comment:
Yeah so this is one part of what i was saying above. Look into the other functions too.